frontend/src/app/app-root/starter/starter.component.ts

```diff
@@ -79,6 +86,28 @@ export class StarterComponent implements OnInit, OnDestroy {
     });
   }
 
+  changePassword(): void {
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+    this.pcs.showPasswordChangeDialog({ id: this.mds.getAuthData()?.id!, name: this.mds.getAuthData()?.displayName! })
+      .subscribe(respOk => {
+        if (!respOk) {
+          const dialog = this.dialog.open(MessageDialogComponent, {
+            width: '400px',
+            data: <MessageDialogData>{
+              title: 'Passwort erfolgreich geändert',
```

Comment:
Dieser Dialog erscheint auch, wenn man in der Passwort-Ändern-Dialog-Box auf "Abbrechen" clickt. Man wird auch dann ausgeloggt. Analog erscheint, wenn man in der Super-Admin-Ansicht das Passwort Ändern will und dann auf Abbrechen clickt die SnackBar "Passwort geändert" .